benchmark/scripts/build_script_helper.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 from __future__ import print_function
```

Comment:
We should drop `__future__`

Reply:
this is intended as just dropping all the shebangs and not doing any compatibility cleanups, just making it easy to test if anything breaks with Python 3 (since all the `__future__` etc. should keep working)

Reply:
Probably shouldn't be merged at all